pkg/debugapi/router.go

```diff
@@ -125,15 +125,18 @@ func (s *Service) newRouter() *mux.Router {
 		"GET": http.HandlerFunc(s.peerBalanceHandler),
 	})
 
-	router.Handle("/settlements", jsonhttp.MethodHandler{
-		"GET": http.HandlerFunc(s.settlementsHandler),
-	})
-
-	router.Handle("/settlements/{peer}", jsonhttp.MethodHandler{
-		"GET": http.HandlerFunc(s.peerSettlementsHandler),
+	router.Handle("/pseudosettlements", jsonhttp.MethodHandler{
```

Comment:
Let's give this variable (user-exposed) a more proper name. We can leave the internal names at pseudosettlements.

suggestion: `timesettlements`

Reply:
done